test/factory/common_factory_tests.dart

```diff
@@ -268,15 +269,11 @@ void refTests<T>(ReactComponentFactoryProxy factory, {void verifyRefValue(dynami
     verifyRefValue(refObject.current);
   });
 
-  _typedCallbackRefTests<T>(factory);
-}
-
-void _typedCallbackRefTests<T>(react.ReactComponentFactoryProxy factory) {
-  if (T == dynamic) {
-    throw ArgumentError('Generic parameter T must be specified');
-  }
-
   group('has functional callback refs when they are typed as', () {
+    if (T == dynamic) {
```

Comment:
I'm missing something here - what `T` is this referencing?

Reply:
`T` is the generic parameter to `refTests`, which represents the expected type of the ref